components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -16,9 +16,17 @@
   display: flex;
   flex-direction: column;
   width: 100%;
+  max-width: 680px;
```

Comment:
why wouldn't we have a single container for the conversation that includes the input box? They have the same width and visually look connected (not to mention semantically)

Reply:
because we have a scroller element which should have a full width

Reply:
👍 Might be better as `conversationContent` then

Reply:
changed